src/main/java/org/folio/rest/impl/ExportConfigurationsImpl.java

```diff
@@ -0,0 +1,196 @@
+          }
+        });
+      } catch (Exception e) {
+        logger.error(e.getMessage(), e);
+        asyncResultHandler.handle(Future.succeededFuture(GetBatchVoucherStorageExportConfigurationsCredentialsByIdResponse
+          .respond500WithTextPlain(Response.Status.INTERNAL_SERVER_ERROR.getReasonPhrase())));
+      }
+    });
+  }
+
+  @Validate
+  @Override
+  public void deleteBatchVoucherStorageExportConfigurationsCredentialsById(String id, String lang, Map<String, String> okapiHeaders,
+      Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    vertxContext.runOnContext((Void v) -> {
+      try {
+        String tenantId = TenantTool.calculateTenantId(okapiHeaders.get(RestVerticle.OKAPI_HEADER_TENANT));
+        PostgresClient pgClient = PostgresClient.getInstance(vertxContext.owner(), tenantId);
+        CQLWrapper cqlWrapper = getCqlWrapperByExportConfigId(id);
+
+        pgClient.delete(EXPORT_CONFIG_CREDENTIALS_TABLE, cqlWrapper, reply -> {
+          try {
+            if (reply.succeeded()) {
+              if (reply.result()
+                .getUpdated() == 0) {
+                asyncResultHandler
+                  .handle(Future.succeededFuture(DeleteBatchVoucherStorageExportConfigurationsCredentialsByIdResponse
+                    .respond404WithTextPlain(Response.Status.NOT_FOUND.getReasonPhrase())));
+              } else {
+                asyncResultHandler.handle(
+                    Future.succeededFuture(DeleteBatchVoucherStorageExportConfigurationsCredentialsByIdResponse.respond204()));
+              }
+            } else {
+              asyncResultHandler.handle(Future.succeededFuture(
+                  DeleteBatchVoucherStorageExportConfigurationsCredentialsByIdResponse.respond500WithTextPlain(reply.cause()
+                    .getMessage())));
+            }
+          } catch (Exception e) {
+            logger.error(e.getMessage(), e);
+            asyncResultHandler.handle(Future.succeededFuture(DeleteBatchVoucherStorageExportConfigurationsCredentialsByIdResponse
+              .respond500WithTextPlain(Response.Status.INTERNAL_SERVER_ERROR.getReasonPhrase())));
+          }
+        });
+      } catch (Exception e) {
+        logger.error(e.getMessage(), e);
+        asyncResultHandler.handle(Future.succeededFuture(DeleteBatchVoucherStorageExportConfigurationsCredentialsByIdResponse
+          .respond500WithTextPlain(Response.Status.INTERNAL_SERVER_ERROR.getReasonPhrase())));
+      }
+    });
+  }
+
+  @Validate
+  @Override
+  public void putBatchVoucherStorageExportConfigurationsCredentialsById(String id, String lang, Credentials entity,
+      Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    if (StringUtils.equals(entity.getExportConfigId(), id)) {
+      PgUtil.put(EXPORT_CONFIG_CREDENTIALS_TABLE, entity, entity.getId(), okapiHeaders, vertxContext,
```

Comment:
Since id is not required field in `Credentials`, it can have null value

Reply:
Good catch